libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/mentions/ResolvedSuggestion.kt

```diff
@@ -8,13 +8,27 @@
 package io.element.android.libraries.textcomposer.mentions
 
 import androidx.compose.runtime.Immutable
+import io.element.android.libraries.designsystem.components.avatar.AvatarData
+import io.element.android.libraries.designsystem.components.avatar.AvatarSize
 import io.element.android.libraries.matrix.api.core.RoomAlias
+import io.element.android.libraries.matrix.api.core.RoomId
 import io.element.android.libraries.matrix.api.room.RoomMember
-import io.element.android.libraries.matrix.api.roomlist.RoomSummary
 
 @Immutable
 sealed interface ResolvedSuggestion {
     data object AtRoom : ResolvedSuggestion
     data class Member(val roomMember: RoomMember) : ResolvedSuggestion
-    data class Alias(val roomAlias: RoomAlias, val roomSummary: RoomSummary) : ResolvedSuggestion
+    data class Alias(
```

Comment:
We will probably want to do the same type of rework for `Member.roomMember` (later!)